src/main/java/core/GdxGame.java

```diff
@@ -11,9 +12,13 @@
 
     public static final int WIDTH = 1024;
     public static final int HEIGHT = 768;
+    public SpriteBatch batch;
 
     @Override
     public void create() {
+
+        this.batch = new SpriteBatch();
```

Comment:
imo SpriteBatch should be a property of the HUD or HUDManager

i'm pretty sure putting all our textures in one SpriteBatch kinda loses the benefits, but we should look into this more

Reply:
Right, moved